optuna/storages/_rdb/storage.py

```diff
@@ -145,6 +145,21 @@ def objective(trial):
         https://docs.sqlalchemy.org/en/13/core/engines.html#sqlalchemy.create_engine.params.
         pool_pre_ping
 
+    .. note::
+        Mainly in a cluster environment, running trials are often killed unexpectedly by the job
+        scheduler. If you want to monitor the trial's life and death, please use the heartbeat
+        mechanism. Set ``heartbeat_interval``, ``grace_period``, and `failed_trial_callback``
+        appropriately according to your use case. For more details, please refer to the
+        `FAQ page`_ and `Example page
+        <https://github.com/optuna/optuna-examples/blob/main/pytorch/pytorch_checkpoint.py>`_.
+
+    .. _FAQ page:
+        https://optuna.readthedocs.io/en/latest/faq.html#can-i-monitor-trials-and-make-them-
```

Comment:
I'm wondering if we could link from here to the corresponding Optuna's version page rather than the latest.

Reply:
I will use `v3.0.0-a1`. Thanks!

Reply:
How about adding a todo note to replace url after the release of v3.0.0?

Reply:
Sorry, i meant to use the Sphinx syntax to link the FAQ section like ttps://github.com/optuna/optuna/blob/444695dfc7a82d054e8f68fc5035426709389c18/optuna/study/study.py#L412. Otherwise, we need to update this version at every version update.

Reply:
I see. My apology for misunderstanding. I made the patch to apply @nzw0301's suggestion:

```diff
diff --git a/docs/source/faq.rst b/docs/source/faq.rst
index 4f03d3fa2..8a264228d 100644
--- a/docs/source/faq.rst
+++ b/docs/source/faq.rst
@@ -534,6 +534,9 @@ However, if you are in the environment where you can not install a client/server

 For more information about 3., see TutorialEasyParallelization_.

+
+.. _heartbeat_monitoring:
+
 Can I monitor trials and make them failed automatically when they are killed unexpectedly?
 ------------------------------------------------------------------------------------------

diff --git a/optuna/storages/_rdb/storage.py b/optuna/storages/_rdb/storage.py
index 88d0816ce..de9433b19 100644
--- a/optuna/storages/_rdb/storage.py
+++ b/optuna/storages/_rdb/storage.py
@@ -82,7 +82,6 @@ def _create_scoped_session(
         session.close()


-# TODO(HideakiImamura): Replace the URL of the FAQ page after the release of v3.0.0.
 class RDBStorage(BaseStorage):
     """Storage class for RDB backend.

@@ -151,7 +150,7 @@ class RDBStorage(BaseStorage):
         If you want to detect a failure of trials, please use the heartbeat
         mechanism. Set ``heartbeat_interval``, ``grace_period``, and `failed_trial_callback``
         appropriately according to your use case. For more details, please refer to the
-        `FAQ page`_ and `Example page
+        :ref:`tutorial <heartbeat_monitoring>` and `Example page
         <https://github.com/optuna/optuna-examples/blob/main/pytorch/pytorch_checkpoint.py>`_.

     .. _FAQ page:
```

@HideakiImamura Could you please apply it? :bow: